src/@ionic-native/plugins/device-feedback/index.ts

```diff
@@ -1,6 +1,14 @@
 import { Injectable } from '@angular/core';
 import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
 
+export interface DeviceFeedbackEnabled {
```

Comment:
Should be called `DeviceFeedbackStatus` @danielsogl

Reply:
fixed